src/services/CweService.ts

```diff
@@ -0,0 +1,80 @@
+import { osimRuntime } from '@/stores/osimRuntime';
+import type { ZodCWEMemberType } from '@/types/zodMitre';
+
+const DATA_KEY = 'CWE:API-DATA';
+const VERSION_KEY = 'CWE:API-VERSION';
+
+interface CweMember {
+  CweID: string;
+}
+interface CweCategory {
+  ID: string;
+  Name: string;
+}
```

Comment:
Since these interfaces are only used here for the network, I'll prefer they have the full path of the data instead of just the end attributes, since we are using `ZodCWEMemberType` for that:


```typescript
interface CweViews {
  Views: Array<{Members: Array<{CweID: string}>}>;
}
interface CweCategories {
  Categories: Array<{ID: string; Name: string}>;
}
```